plugins/lookup/onepassword_raw.py

```diff
@@ -76,18 +76,19 @@
 class LookupModule(LookupBase):
 
     def run(self, terms, variables=None, **kwargs):
-        op = OnePass()
-
-        vault = kwargs.get('vault')
-        op.subdomain = kwargs.get('subdomain')
-        op.username = kwargs.get('username')
-        op.secret_key = kwargs.get('secret_key')
-        op.master_password = kwargs.get('master_password', kwargs.get('vault_password'))
+        vault = kwargs.get("vault")
+        subdomain = kwargs.get("subdomain")
+        domain = kwargs.get("domain", "1password.com")
+        username = kwargs.get("username")
+        secret_key = kwargs.get("secret_key")
+        master_password = kwargs.get("master_password", kwargs.get("vault_password", ""))
```

Comment:
I guess sooner or later it would make sense to switch to proper option handling here (`get_option()` and `set_options()`).

Reply:
I forgot about that method being available for plugins. Thanks for the reminder.